input/intro-notes/StructureDefinition-au-core-medicationrequest-intro.md

```diff
@@ -18,7 +18,7 @@ Conformance in reverse is not guaranteed, i.e. a resource conforming to [Interna
 
 
 ### Profile specific implementation guidance
-- See the [Medicine Information](medicine-information.html) page for guidance
+- See the [Medicine Information](medicine-information.html) page for guidance.
```

Comment:
This comment is related to the information guidance in Medications profile. Add full stop to the end of the information guidance that references the same page.

Reply:
Change made to AU Core Medication.